packages/blockchain-wallet-v4-frontend/src/scenes/TaxCenter/template.tsx

```diff
@@ -106,26 +134,59 @@ const TaxCenter = ({ onGenerateReportClick }: Props) => (
       }
     >
       <Content>
-        <SelectGroup>
-          <SelectLabel size='14px' weight={600} color='black'>
+        <GenerateReport>
+          <SelectGroup>
+            <SelectLabel size='14px' weight={600} color='black'>
+              <FormattedMessage
+                id='scenes.tax.center.card.report.select'
+                defaultMessage='Choose tax year'
+              />
+            </SelectLabel>
+            <StyledSelect
+              value={value}
+              elements={[{ group: '', items: options }]}
+              onBlur={() => {}}
```

Comment:
Can you just remove `onBlur`?

Reply:
I don't know why but the select need the onBlur definition. it's weird but works